### PR TITLE
Releasing TTK 2.0.0b5

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: docs/releases/2.0.0b5.rst [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+This release contains many improvements and bug fixes. While it contains many
+general improvements, it also specifically contains needed changes and
+optimizations for the upcoming `Pootle <http://pootle.translatehouse.org/>`_
+2.8.0 and `Virtaal <http://virtaal.translatehouse.org>`_ releases.
+
+
+2.0.0b5 vs 2.0.0b4
+==================
+
+- Added new Mozilla's l20n format, including converters `l20n2po` and `po2l20n`
```
Comment (dwaynebailey) [inline docs/releases/2.0.0b5.rst]: Can you link these to the doc page?

File: docs/releases/2.0.0b5.rst [hunk 2]
```diff
@@ -0,0 +1,234 @@
+Translate Toolkit 2.0.0b5
+*************************
+
+*Released on 29 August 2016*
+
+This release contains many improvements and bug fixes. While it contains many
+general improvements, it also specifically contains needed changes and
+optimizations for the upcoming `Pootle <http://pootle.translatehouse.org/>`_
+2.8.0 and `Virtaal <http://virtaal.translatehouse.org>`_ releases.
```
Comment (dwaynebailey) [inline docs/releases/2.0.0b5.rst]: Can we try liven up this first paragraph.  Its the first thing anyone sees and its the same boring text very single release.

File: docs/releases/2.0.0b5.rst [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+
+2.0.0b5 vs 2.0.0b4
+==================
+
+- Added new Mozilla's l20n format, including converters `l20n2po` and `po2l20n`
+- Added ical format support for Python 3
+- Mozilla's lang format always outputs last unit followed by trailing newlines
+- ``po2prop`` skips first entry in PO file (:issue:`3463`)
```
Comment (dwaynebailey) [inline docs/releases/2.0.0b5.rst]: Link to command?

**Comment by dwaynebailey** [comment]
lgtm with some minor comments on the RN

File: docs/releases/2.0.0b5.rst (hunk 2 above)
Comment (unho) [inline docs/releases/2.0.0b5.rst]: IMHO it helps giving a consistent look to all release notes. And what it says is still true.

**Pull request merged**